packages/utilities/psammead-locales/moment/ti.js

```diff
@@ -0,0 +1,13 @@
+var moment = require('moment');
+
+moment.defineLocale('yi', {
```

Comment:
```suggestion
moment.defineLocale('ti', {
```

Reply:
Hopefully, this will be the last `yi` to replace :P